osf/utils/users.py

```diff
@@ -0,0 +1,18 @@
+from osf.models import OSFUser
+from django.core.exceptions import ValidationError
+
+def get_or_refresh_confirmation_link(uid):
```

Comment:
This function appears unused, and I'm not sure that it needs to exist, generally.

The deleted/confirmed messages should probably be used in the admin view fn for those cases, though.